docs/Glossary.md

```diff
@@ -34,6 +34,11 @@ Has two meanings:
 
 Connects two [Nodes](#node) in the same area. Contains a set of [Requirements](#requirement) that define whether the [Generator](#generator) and [Solver](#solver) are allowed to cross it.
 
+## Cosmetic Patches
+
+Settings that do not impact the permalink/generation, but can be used when exporting the game to optionally tweak certain things.
```

Comment:
I'd add either in this line, or in the line below some sentence for things that don't give unfair advantages. I.e. Cutscene skips, while not affecting permalink/generation, should *not* be cosmetic

Reply:
Eh, I'll leave this for somewhere else.

But also by definition anything that doesn't affect the Permalink is Cosmetic. These are the only two kind of settings we have. So cutscene skips affect the permalink.
But Echoes has plenty cosmetic patches that can give you disadvantages in racing if disabled.